lib/tarfiles.py

```diff
@@ -145,7 +146,7 @@ def do_tarballs(args: Dict[str, str]) -> None:
         try:
             os.unlink(tf)
         except:  # pylint: disable=bare-except
-            print(f"Notice: unable to remove generated tarfile {tf}")
+            print(f"Notice: unable to remove generated tarfile {tf}")  # type: ignore
```

Comment:
Why the `type: ignore` here?

Reply:
See above comment about possible mypy open bugs.